library/fluentd

```diff
@@ -3,13 +3,25 @@ Maintainers: Masahiro Nakagawa <repeatedly@gmail.com> (@repeatedly),
 GitRepo: https://github.com/fluent/fluentd-docker-image.git
 
 # Alpine images
-Tags: v1.16.2-1.1, v1.16-1, latest
+Tags: v1.16.6-1.0, v1.16-1
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: a4dd65768ec1819574e570716955276c9089326a
+GitCommit: ddb5019b433339f9eef991ca373445deb6e4d2b1
```

Comment:
From the failing test:
```console
 - fluentd:
   - commit ddb5019b433339f9eef991ca373445deb6e4d2b1:
     - is not in the specified ref GitFetch: refs/heads/master
```

It looks like this [commit](https://github.com/fluent/fluentd-docker-image/commit/ddb5019b433339f9eef991ca373445deb6e4d2b1) is in the `v1.16` branch, so these two 16.x blocks would need a `GitFetch: refs/heads/v1.16` to reflect that.

Reply:
Thanks! I've overlooked it.